wazo_confd/plugins/user/schema.py

```diff
@@ -369,3 +373,4 @@ class UserLinePutSchema(LineSchema):
 class UserListItemSchemaPut(UserListItemSchema):
     lines = Nested('UserLinePutSchema', many=True)
     agent = Nested('UserAgentSchema', allow_none=True)
+    incalls = Nested('UserIncallPutSchema', many=True)
```

Comment:
This is exposing the extensions in the /user API?

Reply:
This is not directly related. I found an issue when we try to update a user with their incalls (the call id is ignored). So instead of created a dedicated PR for that and lose time for the other PR to be reviewed, I included this fix in the current PR :-)

Reply:
That sounds like a significant issue in the public API though, would be worth describing the issue somewhere(jira) to anticipate reporting, and point to this PR.

Reply:
Done :)